### PR TITLE
Various post 0.15 fixes

### LINKED ISSUE #238: The background of the RTD version picker is the wrong colour
### Describe the bug

clipboard

### To Reproduce

_No response_

### Screenshots

_No response_

### System Details

_No response_

### Installation method

_No response_

### LINKED ISSUE #239: The search bar has a weird border
### Describe the bug

clipboard

### To Reproduce

_No response_

### Screenshots

_No response_

### System Details

_No response_

### Installation method

_No response_

### PR DESCRIPTION
**Pull request by Cadair**
fixes #239 
fixes #238

### CODE REVIEW

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css [hunk 1]
```diff
@@ -287,6 +287,28 @@ img.logo__image {
...
+
+.search-button__wrapper.show form.bd-search:focus-within {
+  border: 2px solid var(--pst-color-accent);
+}
+
+html[data-theme="light"] .search-button-field:hover {
+  background-color: var(--sst-light-color);
+}
+
+.search-button-field:hover {
```
Comment (Cadair) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:308]:
```suggestion
html[data-theme="dark"] .search-button-field:hover {
```

**Review by nabobalis — APPROVED** [review]

File: src/sunpy_sphinx_theme/theme/sunpy/components/navbar_center.html [hunk 2]
```diff
@@ -29,5 +29,10 @@
         {% endif %}
       {%- endfor %}
     {% endif %}
+    {% if theme_external_links %}
+      {%- for external_link in theme_external_links %}
+          <li class="nav-item ms-2"><a class="nav-link nav-external" role="button" href="{{ external_link['url'] }}">{{ external_link['name'] }}</a></li>
+      {%- endfor %}
+    {% endif %}
```
[the remark this answers is not in the document]
Comment (Cadair) [reply src/sunpy_sphinx_theme/theme/sunpy/components/navbar_center.html:36]: Nope, I must have forgotten about it when I implemented the header.

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css [hunk 3]
```diff
@@ -232,7 +232,7 @@ html[data-theme="light"] .bd-sidebar-primary .theme-switch-button span {
   --bs-dropdown-border-radius: 0rem;
   background-color: var(--sst-header-background);
   box-shadow: none;
-  margin-top: 0.85rem;
+  margin-top: 0.86rem;
```
Comment (alasdairwilson) [inline src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:235]: haha 0.01 rem,

File: docs/conf.py [hunk 4]
```diff
@@ -68,6 +68,16 @@
     "reproject": ("https://reproject.readthedocs.io/en/stable/", None),
 }
 html_theme = "sunpy"
+
+html_theme_options = {
+    "footer_links": [
+        ("Google", "https://google.com", 3),
+        ("DDG", "https://duckduckgo.com", 3),
```
[the remark this answers is not in the document]
Comment (Cadair) [reply docs/conf.py:75]: these are just for testing customising this (which didn't work before).

If you don't set it you get the standard ones on all the sunpy.org sites.

File: src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css (hunk 3 above)
Comment (Cadair) [reply src/sunpy_sphinx_theme/theme/sunpy/static/sunpy_style.css:235]: it was like 1px out at normal zoom for me lol

**Pull request merged**